### PR TITLE
Improvements across.

### PR DESCRIPTION
**Pull request by RamonGilabert**

### CODE REVIEW

File: Demo/ImagePickerDemo/Podfile.lock [hunk 1]
```diff
@@ -6,9 +6,9 @@ DEPENDENCIES:
...
 EXTERNAL SOURCES:
   ImagePicker:
-    :path: "../../"
+    :path: ../../
 
 SPEC CHECKSUMS:
   ImagePicker: 32becfa25b8e9179e60c45411b577340d35e3e32
 
-COCOAPODS: 0.39.0
+COCOAPODS: 0.39.0.beta.4
```
Comment (zenangst) [inline Demo/ImagePickerDemo/Podfile.lock]: giphy-59

Comment (RamonGilabert) [inline Demo/ImagePickerDemo/Podfile.lock]: I'm a renegade.

Comment (zenangst) [inline Demo/ImagePickerDemo/Podfile.lock]: Rebel without a cause :sunglasses:

**Comment by zenangst** [comment]
@RamonGilabert :sunrise:

**Pull request merged**